python/lsst/sdm/tools/cli.py

```diff
@@ -0,0 +1,81 @@
+# This file is part of sdm_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import zipfile
+from pathlib import Path
+
+import click
+
+from . import __version__
+from . import build_datalink_metadata as _build_datalink_metadata
+
+__all__ = ["cli"]
+
+
+@click.group()
+@click.version_option(__version__)
+@click.pass_context
+def cli(ctx: click.Context) -> None:
+    """SDM Tools Command Line Interface"""
+    ctx.ensure_object(dict)
+
+
+@cli.command("build-datalink-metadata", help="Build Datalink metadata from Felis YAML files")
+@click.argument("files", type=click.Path(exists=True), nargs=-1, required=True)
```

Comment:
You don't want to support ResourcePath for remote access to felis files?

Reply:
None of the Felis command-line tools support resource paths for reading schemas, though I do like the idea.

Reply:
I'll address this on a separate ticket where Felis is updated, too. I don't think that functionality should be introduced here if it is not available in Felis, which has the main command-line tool for working with the YAML schema files.